apps/rotm/index.js

```diff
@@ -48,7 +49,13 @@ module.exports = {
     },
     '/check-your-report': {
       prereqs: ['/image'],
-      behaviours: [require('hof-behaviour-summary-page'), 'complete', caseworkerEmailer, checkReportBackLink],
+      behaviours: [
+        require('hof-behaviour-summary-page'),
+        'complete',
+        caseworkerEmailer,
+        checkReportBackLink,
+        removeImage
+        ],
```

Comment:
@sulthan-ahmed This square brace should be vertically inline with the `b` of the word `behaviours` above it